web/src/engine/websites/VortexScans.ts

```diff
@@ -11,4 +17,12 @@ export default class extends VTheme {
     public override get Icon() {
         return icon;
     }
+
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
+        const scripts = await FetchCSS<HTMLScriptElement>(new Request(new URL(chapter.Identifier, this.URI)), 'script:not([src])');
```

Comment:
I think it is not necessary to filter the scripts, the RegExp can probably directly be used on the plain HTML text `await (await Fetch()).text()`